examples/mysql-query-column-mapping/mysql_query_column_mapping.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/io;
+import ballerinax/mysql;
+import ballerina/sql;
+
+// Defines a record to load the query result.
+type Customer record {|
+    @sql:Column {name: "customer_id"}
+    int customerId;
+    @sql:Column {name: "last_name"}
+    string lastName;
+    @sql:Column {name: "first_name"}
+    string firstName;
+|};
+
+public function main() returns error? {
+    // Runs the prerequisite setup for the example.
+    check initialize();
+
+    // Initializes the MySQL client. The `mysqlClient` can be reused to access database throughout the application execution.
+    mysql:Client mysqlClient = check new (user = "root",
+            password = "Test@123", database = "CUSTOMER");
+
+    // Query table with a condition.
+    stream<Customer, error?> resultStream =
+            mysqlClient->query(`SELECT * FROM Customers;`);
+
+    // Iterate the result stream.
+    check from Customer customer in resultStream
+        do {
+            io:println(`Customer details: ${customer}`);
+        };
+
+    // Closes the stream to release resources.
```

Comment:
```suggestion
    // Closes the stream to release the resources.
```

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)